src/sentry/api/serializers/base.py

```diff
@@ -35,8 +41,10 @@ def wrapped(cls):
 
 
 class Serializer(object):
-    def __call__(self, *args, **kwargs):
-        return self.serialize(*args, **kwargs)
+    def __call__(self, obj, attrs, user):
+        if not obj:
```

Comment:
I originally did it inside the loop in serialize, like:

`[serializer(o, attrs=attrs.get(o, {}), user=user) if o else o for o in objects]`

But that seemed worse.

What's wrong with this here?